src/app.js

```diff
@@ -0,0 +1,111 @@
+import axios from "axios";
+async function findCountry(){
+    const BASE_URI = 'https://restcountries.com/'
+    const ENDPOINT = 'v2/name/netherlands'
+    try{
+        const response = await axios.get(BASE_URI + ENDPOINT);
+        const {data:countries} = response;
+         console.log(countries[0]);
+         console.log(countries[0].name);
+         console.log(countries[0].population);
+        // console.log('test')
+        const listOfCountries = document.getElementById('country-list');
+        //console.log(listOfCountries);
```

Comment:
Hier had ik zelf nog spreading toegepast, zodat je niet overal `countries[0].` voor had hoeven zetten:
```suggestion
        //console.log(listOfCountries);  
        const { flags: { png }, name, population, subregion, capital, currencies, languages } = countries[0];
```

Reply:
ok.